index.html

```diff
@@ -35,6 +35,125 @@ <h2 class="greeting-intro">
             <li><a href=""><img class="social" src="assets/img/github.png" alt="github"/></a></li>
             <li><a href=""><img class="social" src="assets/img/angellist.png" alt="angellist"/></a></li>
```

Comment:
Kindly. I recommend you to put # in the A links so that it does not produce any error if someone wants to click on them

Reply:
The # has been added to the links. Thanks 👍